package.json

```diff
@@ -86,22 +86,22 @@
     "hapi": "^16.6.2",
     "hat": "0.0.3",
     "ipfs-api": "^21.0.0",
-    "ipfs-repo": "^0.19.0",
+    "ipfs-repo": "~0.20.0",
```

Comment:
@dryajov even with https://github.com/ipfs/js-ipfsd-ctl/pull/243#issuecomment-388975434, the teardown is just used for testing. It should never be a regular dependency nor be a util from src.

Reply:
@diasdavid it's not only used in tests, we use it the cleanup method as well - https://github.com/ipfs/js-ipfsd-ctl/blob/b7ef098eab862de26cd723acd0c9ce598ff725c4/src/ipfsd-in-proc.js#L181

---

AFAIK, when calling `ipfs.stop()`, the repo doesn't get cleaned up, hence we need something to be able to purge it from the browsers storage (indexDB, websql, etc...), not doing that would give a false sense of the repo being cleaned up/deleted when its not, and doing it by hand would possibly lead us to deal with browser inconsistencies and other corner cases that are most likely taken care already by `ipfs-repo`. 

I might be missing something in my assessment, would be happy to hear otherwise 👍

Reply:
OK, slight correction - seems like the only place we would need `ipfs-repo` would be in https://github.com/ipfs/js-ipfsd-ctl/blob/21b9850ae35e389736b6e43be5470fac0b8cc063/src/factory-in-proc.js#L144

I'll take a closer look how to better handle that piece, once thats out it should be easy to remove `ipfs-repo`.